test/lib/cable_ready/view_helper_test.rb

```diff
@@ -3,8 +3,6 @@
 require "test_helper"
 
 class CableReady::ViewHelperTest < ActionView::TestCase
-  include CableReady::Broadcaster
-
```

Comment:
We probably want to include the `CableReady::ViewHelper` instead here

Reply:
It is included. Verified with a failing test